word_guess.rb

```diff
@@ -0,0 +1,159 @@
+    puts "\nOh no! Another bunny is coming to your yard!"
+      require 'colorize'
+      second_bunny_one = "\n=====================".colorize(:cyan)
+      second_bunny_two = "\n   ()__()     ()_()  "
+      second_bunny_thr = "\n  (=o.o=)     (^+^)  "
+      second_bunny_for = "\n  (w)_(w)*  *(')_(') "
+      second_bunny_fiv = "\n=====================".colorize(:cyan)
+      puts second_bunny_one + second_bunny_two + second_bunny_thr + second_bunny_for + second_bunny_fiv
+      puts "You have #{count} incorrect guesses before bunnies take over!"
+    elsif count == 2
+    puts "\nOh no! Another bunny is coming to your yard!"
+      require 'colorize'
+      third_bunny_one = "\n===============================".colorize(:light_blue)
+      third_bunny_two = "\n  (')_(')     ()_()     ()_()  "
+      third_bunny_thr = "\n  ('o+o')     (^+^)     (O.o)  "
+      third_bunny_for = "\n  (@)_(@)*  *(')_(')  *(')(')  "
+      third_bunny_fiv = "\n===============================".colorize(:light_blue)
+      puts third_bunny_one + third_bunny_two + third_bunny_thr + third_bunny_for + third_bunny_fiv
+      puts "You have #{count} incorrect guesses before bunnies take over!"
+    elsif count == 1
+      puts "\nOh no! Another bunny is coming to your yard!"
+      puts "You have #{count} incorrect guesses before bunnies take over!"
+      require 'colorize'
+      fourth_bunny_one = "\n========================================".colorize(:cyan)
+      fourth_bunny_two = "\n  (')_(')     ()_()     ()_()   ()_()   "
+      fourth_bunny_thr = "\n  ('o+o')     (^+^)     (O.o)   (-.-)   "
+      fourth_bunny_for = "\n  (@)_(@)*  *(')_(')  *(')(')  (')(')*  "
+      fourth_bunny_fiv = "\n========================================".colorize(:cyan)
+      puts fourth_bunny_one + fourth_bunny_two + fourth_bunny_thr + fourth_bunny_for + fourth_bunny_fiv
+      puts "You have #{count} incorrect guesses before bunnies take over!"
+    end
+  end
+
+
+  #WIN
+  if letter_array & all_guesses == letter_array
+    require 'colorize'
+    puts "Your word was #{letter_array.join.upcase}!"
+    puts "You kept the bunnies away! YOU WIN! :)".colorize(:green)
+    flower0 = "\n===============".colorize(:cyan)
+    flower1 = "\n    _ _        ".colorize(:yellow)
+    flower2 = "\n  _{ ' }_      ".colorize(:yellow)
+    flower3 = "\n { `.!.` }     ".colorize(:yellow)
+    flower4 = "\n ',_/Y _,'     ".colorize(:yellow)
+    flower5 = "\n   {_,_}       ".colorize(:yellow)
+    flower6 = "\n     |         ".colorize(:green)
+    flower7 = "\n   ( |  /)     ".colorize(:green)
+    flower8 = "\n    (| //      ".colorize(:green)
+    flower9 = "\n     |//       ".colorize(:green)
+    flower10 = "\n     |/  //    ".colorize(:green)
+    flower11 = "\n ^^^^^^^^^^^^ ".colorize(:green)
+    flower12 = "\n==============".colorize(:cyan)
+    puts flower0 + flower1 + flower2 + flower3 + flower4 + flower5 + flower6 + flower7 + flower8 + flower9 + flower10 + flower11 + flower12
```

Comment:
One option that you have to do this as well is to create an array for each piece of this ASCII art. Then you can use `flower.join("")` here rather than concatenating each individual string.